tests/unit/Term/AliasGroupListTest.php

```diff
@@ -170,20 +171,13 @@ public function testGivenInvalidLanguageCode_removeByLanguageThrowsException( $l
 	}
 
 	public function testGivenEmptyGroups_constructorRemovesThem() {
-		$enGroup = new AliasGroup( 'en', array( 'foo' ) );
-
 		$list = new AliasGroupList( array(
 			new AliasGroup( 'de' ),
-			$enGroup,
-			new AliasGroup( 'en' ),
-			new AliasGroup( 'nl' ),
-		) );
-
-		$expectedList = new AliasGroupList( array(
+			new AliasGroup( 'en', array( 'foo' ) ),
 			new AliasGroup( 'en' ),
 		) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertEquals( new AliasGroupList(), $list );
```

Comment:
What are you missing?

This test does not test what it says it tests.

1. The name suggests this is about the removal of empty groups in the constructor.
2. The assertion compares two objects that happen to be equal. Could be because they are both empty, could be because they both contain an empty group for `en`.

I found this an obvious mistake with an obvious fix when I uploaded this patch. So again, what are you missing?